extra-web/baidupcs-go/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=baidupcs-go
+PKGSEC=net
+PKGDES="A baidupcs unofficial client"
```

Comment:
Change this to "An unofficial client for BaiduPCS or Baidu Yun".

Reply:
> Change this to "An unofficial client for BaiduPCS or Baidu Yun".

ok